tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -113,6 +115,10 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
 
         $kernelProphecy = $this->prophesize(KernelInterface::class);
         $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
+        $containerBuilderProphecy = $this->prophesize(ContainerBuilder::class);
+
+        $kernelProphecy->getContainer()->willReturn($containerBuilderProphecy);
+        $containerBuilderProphecy->has(Argument::any())->willReturn(true);
```

Comment:
We can limit it to the specific values we need here.
On 27 Jun 2016 21:50, "Kévin Dunglas" <notifications@github.com> wrote:

> In tests/Bridge/Symfony/Routing/ApiLoaderTest.php
> <https://github.com/api-platform/core/pull/586#discussion_r68579198>:
>
> > @@ -113,6 +115,10 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
> >
> >          $kernelProphecy = $this->prophesize(KernelInterface::class);
> >          $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
> > +        $containerBuilderProphecy = $this->prophesize(ContainerBuilder::class);
> > +
> > +        $kernelProphecy->getContainer()->willReturn($containerBuilderProphecy);
> > +        $containerBuilderProphecy->has(Argument::any())->willReturn(true);
>
> Argument::type('string') indeed
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/api-platform/core/pull/586/files/6b32716eb4e97b5aadd594a29fb7ee625b0490a8#r68579198>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAhf647GnFhS_HyQaZz6cQNKgae5Pvahks5qP9VDgaJpZM4I-_Cx>
> .
>